src/ripples/focus-ripple.jsx

```diff
@@ -73,7 +74,7 @@ const FocusRipple = React.createClass({
       transition: Transitions.easeOut(pulsateDuration + 'ms', 'transform', null, Transitions.easeInOutFunction),
     }, innerStyle);
 
-    return <div ref="innerCircle" style={this.prepareStyles(innerStyles)} />;
+    return <div ref="innerCircle" style={prepareStyles(innerStyles)} />;
```

Comment:
`innerStyle` is external. might it be safer to `Object.assign({},...)` this too?